### ISSUE TITLE
[SF][a11y][ui5-date-picker] Focus outline of year or month is missed after some date is selected

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#5644]
### **Bug Description**
We found the focus outline of year and month in `ui5-date-picker` is missed after we select some date via component. The current year and current month have the focus outline when the picker is empty.
![2022-08-09_14-15-42](https://user-images.githubusercontent.com/1867889/183577928-17f969b0-252d-442c-9474-e23873941720.png)

But if we select some value and then open the year or month picker again, the outline is lost.
![2022-08-09_14-15-57](https://user-images.githubusercontent.com/1867889/183578047-65b9c0a1-dc1f-40fc-b4ed-adb759e6cd7b.png)

### **Expected Behavior**
Current month or year should have the focus outline after some date is selected.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/DatePicker/?sap-ui-theme=sap_horizon&isrtl=false
2. Open any date picker in the demo page, use shift-tab to focus on the year picker button and press space key to open it.
3. The current year is focused and has focus outline on it.
4. Select some date and repeat the steps above, the current year does not have focus outline on it.

### **Context**
- UI5 Web Components version: 1.6.0
- OS/Platform: MacOS
- Browser: Edge
- Affected component: ui5-date-picker

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-topic-b  the issue is reproducible  when the date picker is closed and then opened again can you take a look whether this is the expected behaviour?
